src/components/number/number.css

```diff
@@ -0,0 +1,41 @@
+@import '../../styles/colors-v2.css';
+@import "../../styles/spacing.css";
+@import "../../styles/typography.css";
+
+.input {
+  border: 1px solid var(--mds-grey-12);
+  border-radius: 3px;
+  box-sizing: border-box;
+  font-family: var(--mavenlink-type-font-family); /* stylelint-disable-line mds/typography */
```

Comment:
Do we need to skip the linter here?

Reply:
Yeah the linter gets mad without it for some reason

Reply:
Why is that? Do we want to be using the correct typography variables?

Reply:
@cincospenguinos ?

Reply:
I'm not entirely sure. That's something we'll need to explore